lib/galaxy/config/sample/object_store_conf.xml.sample

```diff
@@ -1,125 +1,206 @@
 <?xml version="1.0"?>
+<!--
+    Sample Object Store configuration file
+
+    There should only be one root <object_store> tag, multiple are shown here to
+    show different configuration options. Any object store can be used as
+    backends to the distributed and hierarchical object stores (including
+    distributed and hierarchical themselves).
+-->
+
+<!--
+    Sample Disk Object Store
+
+    This mirrors the default configuration if there is no object store
+    configuration file. The default uses the values of file_path, new_file_path,
+    and job_working_directory in galaxy.yml).
+-->
+<object_store type="disk" store_by="uuid">
+    <files_dir path="database/objects"/>
+    <extra_dir type="temp" path="database/tmp"/>
+    <extra_dir type="job_work" path="database/jobs_directory"/>
+</object_store>
+
+<!--
```

Comment:
Can we comment these to keep the file valid ?

Reply:
Yes